packages/miniflare/src/plugins/core/index.ts

```diff
@@ -712,7 +716,11 @@ export function getGlobalServices({
 		{ name: CoreBindings.JSON_LOG_LEVEL, json: JSON.stringify(log.level) },
 		{
 			name: CoreBindings.SERVICE_USER_FALLBACK,
-			service: { name: getUserServiceName(fallbackWorkerName) },
+			service: {
```

Comment:
IMHO this decision should be moved to where `getGlobalServices` is [called](https://github.com/cloudflare/workers-sdk/blob/main/packages/miniflare/src/index.ts#L1249) and should not be an implementation detail of this fn

Reply:
I don't think this comment has been addressed. Marking as unresolved